docs/reference/commandline/cli.md

```diff
@@ -45,6 +45,7 @@ by the `docker` command line:
   unsuitable for Docker.
 * `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
+* `DOCKER_NOTARY` When set Docker uses notary to sign and verify images.
```

Comment:
This is so eventually we can make trusted the default and not have it be counterintuitive.

Otherwise all options sound weird when they are turned on by default. There was a alot of back and forth on this already and I think I am still pro- the current approach.

On Jul 13, 2015, at 08:37, Dave Tucker <notifications@github.com> wrote:

In docs/reference/commandline/cli.md:

> @@ -45,6 +45,7 @@ by the `docker` command line:
>    unsuitable for Docker.
>  * `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
>  * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
> +* `DOCKER_NOTARY` When set Docker uses notary to sign and verify images.
--untrusted=false feels a little like a double negative. E.g it is not un-trusted. Therefore it might be difficult for non-native english speakers to divine the expected behaviour. It's simpler to say it is trusted. I appreciate trusted is a loaded term, so maybe it's not the right word to use. Per icecrime's comment, I agree that we should use something more generic. Perhaps splitting the two workflows (signing and verification) would make sense - e.g DOCKER_IMAGE_VERIFY and DOCKER_IMAGE_SIGN

—
Reply to this email directly or view it on GitHub.